src/graphql/scalars/json.ts

```diff
@@ -14,10 +14,13 @@ export const JSONScalar = new GraphQLScalarType({
     return value;
   },
   parseLiteral(node) {
-    if (node.kind !== Kind.STRING) {
-      throw new UserInputError("Invalid type.");
+    if(node.kind === Kind.OBJECT) {
+      return {};
```

Comment:
Hmm 🤔, this can result in silent dataloss, maybe it's time replace `JSONScalar` by [graphql-type-json](https://github.com/taion/graphql-type-json)'s `GraphQLJSONObject`.